invenio_app_rdm/theme/views.py

```diff
@@ -30,6 +30,7 @@ def create_blueprint(app):
     )
 
     blueprint.add_url_rule(routes["index"], view_func=index)
+    blueprint.add_url_rule(routes["robots"], view_func=robots)
```

Comment:
Can we register this only if we are in development mode i.e `Flask_ENV=devleopment` or `DEBUG=True`? For production/containerized versions we need to implement a rewrite rule on nginx level and use the static file.

Reply:
I think the nginx config will take precedence, so it won't reach the Python app ever.

Reply:
we said maybe to disable that in production setup, just to make sure that this is not available and if users want then they configure nginx... We have a ticket already to add the nginx rule in all our deployment related instances...Do you find any caveat having the above only in development?

Reply:
It's more in the sense that e.g. `/static/` is not disabled in production mode, so if something was wrong in the `nginx` config we wouldn't get a warning/error.

In general, we don't have many places where we disable/enable features based on `debug`, and specifically for `/robots.txt` the impact is not that big (e.g. on Zenodo we had about 2-3req/min to the endpoint).